aten/src/ATen/cuda/CUDAContext.h

```diff
@@ -10,6 +10,10 @@
 #include <cusolverDn.h>
 #endif
 
+#if defined(USE_ROCM) && USE_ROCM && ROCM_VERSION >= 50200
```

Comment:
Why is this preprocessor check different from the one you added to CUDABlas.cpp?

Reply:
Are we supposed to handle `USE_ROCM=0`? If we are I have to fix the CUDABlas.cpp which only checks `#ifdef USE_ROCM`

Reply:
Resolved with standard practice `#if defined(USE_ROCM)`